microfs.py

```diff
@@ -251,11 +258,22 @@ def get(filename, target=None, serial=None):
     if target is None:
         target = filename
     commands = [
-        "from microbit import uart",
+        "\n".join([
+            "try:",
+            " from microbit import uart as u",
+            "except ImportError:",
+            " try:",
+            "  from machine import UART",
+            "  u = UART(0, {})".format(SERIAL_BAUD_RATE),
+            " except Exception:",
+            "  try:",
+            "   from sys import stdout as u",
+            "  except Exception:",
+            "   raise Exception('Could not find UART module in device.')"]),
```

Comment:
First import is for the micro:bit, so we can expect an `ImportError` in other boards.

The second import methods is for esp8266/esp32 MicroPython, however depending on the port the line `UART(0, 115200)`, sometimes works and sometimes throws an exception, depends on if the port has locked the peripheral or default pins, which is why catch all exceptions here to do one last attempt.

The last attempt using sys works on MicroPython esp8266, esp32 and CircuiPython (v3).